main.go

```diff
@@ -27,8 +27,8 @@ var (
 
 func main() {
 	pps := plugin.NewSet()
-	pps.RegisterProvisioner("chef-client", new(chefclient.Provisioner))
-	pps.RegisterProvisioner("chef-solo", new(chefsolo.Provisioner))
+	pps.RegisterProvisioner("client", new(chefclient.Provisioner))
+	pps.RegisterProvisioner("solo", new(chefsolo.Provisioner))
```

Comment:
good catch, we probably should delete releases before merging this then.

Reply:
Are you sure? I think this will cause the checksum problem.

Reply:
(because the old binary will need users to use for example the `chef-chef-client` provisioner)

Reply:
(if we delete the tag, the checksums will still exist but packer init won't ever try to install these.)